code/obj/item/gun/ammo.dm

```diff
@@ -1209,6 +1209,19 @@
 	desc = "A box containg a single inert meowitzer. It appears to be softly purring. Wait is that a cat?"
 	ammo_type = new/datum/projectile/special/meowitzer/inert
 
+/obj/item/ammo/bullets/foamdarts
+	sname = "foam darts"
+	name = "foam dart box"
+	icon_state = "foamdarts"
+	icon_empty = "foamdarts-0"
+	amount_left = 40
+	max_amount = 40
+	caliber = 0.857 // I just hit random numbers on my keyboard
```

Comment:
Small nitpick, the diameter of a Nerf dart is about .393 inches, that could be appropriate as a caliber.

Reply:
You did what you wanted!